CHANGELOG.md

```diff
@@ -16,6 +16,9 @@
 ### Linting
 
 * Adjusted linting to enable `patch` branches from being tested
+* `.travis.yml` now optional, `.circle.yml` and `Singularity` must not be present
```

Comment:
@MaxUlysse might not want that as he's using CircleCI for generating Docker images for Sarek.

Reply:
<img width="604" alt="Bildschirmfoto 2019-12-11 um 13 32 39" src="https://user-images.githubusercontent.com/2359510/70621811-b7681300-1c1a-11ea-94ff-dde6ef570110.png">

Reply:
ok added circle also as optional